vispy/app/backends/_wx.py

```diff
@@ -376,6 +376,11 @@ def _vispy_get_size(self):
         w, h = self.GetClientSize()
         return w, h
 
+    def _vispy_get_physical_size(self):
+        lsize = self.GetClientSize()
+        ratio = self.GetContentScaleFactor()
+        return round(lsize[0] * ratio + 0.01), round(lsize[1] * ratio + 0.01)
```

Comment:
I'm not 100% sure, but this may need to be integers...or at least they are supposed to be I think.